bokehjs/src/lib/models/widgets/markup.ts

```diff
@@ -12,6 +13,17 @@ export abstract class MarkupView extends WidgetView {
 
   protected markup_el: HTMLElement
 
+  get provider(): MathJaxProvider {
+    return default_provider
+  }
+
+  override async lazy_initialize() {
+    await super.lazy_initialize()
+
+    if (this.provider.status == "not_started")
+      await this.provider.fetch()
```

Comment:
You still need to implement the rest of the protocol, the same as in `MathText`. This `await` doesn't guarantee that, e.g. a CDN bundle is loaded at this point (regardless whether we actually support CDN or not).